src/components/MerchantsPage/gam/CampaignListItem.tsx

```diff
@@ -256,25 +255,25 @@ const ImagesContainer = styled.span`
   }
 `;
 
-const CampaignImage = styled.img`
+const CampaignImageContainer = styled.div`
   height: 240px;
   width: 240px;
 
   @media (${tabletScreens}) {
-    max-height: 100px;
+    height: 100px;
     width: 100%;
   }
 `;
 
+const CampaignImage = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
```

Comment:
```suggestion
border-radius: 5px;
`;
```

Reply:
Good eye! Looks much better with the border radius
<img width="530" alt="Screen Shot 2020-09-04 at 9 14 22 PM" src="https://user-images.githubusercontent.com/2313868/92293713-a4594980-eef3-11ea-8ffe-01a03fceab53.png">